spec/support/stub_bibdata.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+module BibdataStubbing
```

Comment:
👍  This is a lot simpler than VCR, and in particular, I'm looking forward to having human-readable sample records on disk instead of VCR's gnarly raw captures.